packages/netlify-cms-core/src/actions/config.ts

```diff
@@ -376,12 +376,13 @@ export function parseConfig(data: string) {
 }
 
 async function getConfigYaml(file: string, hasManualConfig: boolean) {
-  const response = await fetch(file, { credentials: 'same-origin' });
-  if (!response.ok || response.status !== 200) {
```

Comment:
`!response.ok` already means `response.status !== 200`. We are looking for `200` status code.
Also `fetch` can throw so we would like to handle that case as well

Reply:
Could you give an example of fetch throwing? Maybe this should be covered by unit test then.

Reply:
`fetch('http://non-existing-host.com')` should throw.
I've added some tests to this logic in https://github.com/netlify/netlify-cms/pull/4950/commits/8150286cea83f9aed2720fbb85239c316ffd537e

Reply:
Awesome 👍